x-pack/plugins/spaces/index.ts

```diff
@@ -138,10 +139,10 @@ export const spaces = (kibana: Record<string, any>) =>
       const spacesAuditLogger = new SpacesAuditLogger(config, new AuditLogger(server, 'spaces'));
 
       server.expose('spacesClient', {
-        getScopedClient: (request: Record<string, any>) => {
+        getScopedClient: (request: Request) => {
           const adminCluster = server.plugins.elasticsearch.getCluster('admin');
           const { callWithRequest, callWithInternalUser } = adminCluster;
-          const callCluster = (...args: any[]) => callWithRequest(request, ...args);
+          const callCluster = callWithRequest.bind(null, request);
```

Comment:
Since this method technically comes off the `adminCluster` value, we should probably keep the binding in place if possible:

```js
const callCluster = callWithRequest.bind(adminCluster, request);
```